[ADD] 14.0 base rest abstract attachment (#242)
OCA/rest-framework: base_rest_abstract_attachment/readme/DESCRIPTION.rst

> This module is a technical module aiming to be used in other modules.

Reviewer: ```suggestion
Technical module to be used as a base for other modules.
```
